data/relup_bases.eterm

```diff
@@ -0,0 +1,31 @@
+%% -*- mode: erlang; -*-
+
+#{<<"4.4.0">> =>
+      #{from_versions => [],otps => [<<"24.1.5-3">>,<<"23.3.4.9-3">>]},
```

Comment:
please delete otp 23 from the list.
we have produced packages on it, but never really announced it
I don't think anyone is using it